test/specs/elements/Container/Container-test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import Container from 'src/elements/Container/Container'
+import * as common from 'test/specs/commonTests'
+
+describe('Container', () => {
+  common.isConformant(Container)
+  common.rendersChildren(Container)
+  common.hasUIClassName(Container)
+
+  common.propKeyOnlyToClassName(Container, 'text')
+  common.propKeyAndValueToClassName(Container, 'aligned')
+  common.propKeyOnlyToClassName(Container, 'fluid')
+
+  it('renders a <div /> element', () => {
+    shallow(<Container />)
+      .should.have.tagName('div')
+  })
+
+  it('does not have aligned class when justified', () => {
+    shallow(<Container aligned='justified' />)
+      .should.not.have.className('aligned')
+  })
```

Comment:
After updating this PR and getting the new tools merged in #285, remove this test:

```diff
-  it('does not have aligned class when justified', () => {
-    shallow(<Container aligned='justified' />)
-      .should.not.have.className('aligned')
-  })
```

Then, replace line 12 above:
```diff
-common.propKeyAndValueToClassName(Container, 'aligned')
+common.implementsAlignedProp(Container)
```